pkg/controllers/edgeconnect/controller.go

```diff
@@ -429,21 +440,25 @@ func (controller *Controller) getEdgeConnectIdFromClientSecret(ctx context.Conte
 
 	id, err := k8ssecret.ExtractToken(&secret, consts.KeyEdgeConnectId)
 	if err != nil {
+		log.Debug("unable to extract EdgeConnect tokens", "error", err)
 		return "", errors.WithStack(err)
 	}
 
+	log.Debug("exit getEdgeConnectIdFromClientSecret", "id", id)
```

Comment:
do we want to be this detailed?

also if we want to do this then we should have a `start myFunc....` and a corresponding `end myFunc...` for our more important functions, but it still feels overkill for me